style.css

```diff
@@ -0,0 +1,667 @@
+body {
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: gray;
+}
+
+/* Registration page style */
+
+#registration_page {
+  background-color: #F0F8EC;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 844px;
+  width: 390px;
```

Comment:
I think generally you have a log of explicit `height` and `width` definitions that you probably don't need. divs for the most part will fit to the size of the content (which is mostly what you want), and if you want to add extra spacing you can use `padding` or `flex`. setting `height` and `width` explicitly makes your page less responsive and while you may generally work on the same device with the same screen size, someone elses computer screen/phone screen/browser window probably won't be the exact same size

Reply:
Also along that line, I think you'd end up having more class based css selectors instead of id based selectors. Because ids are unique, and css selector on id indicates there's only a single element with that particular style, but in our apps we want to be reusing a lot of style. Like its not that every button is unique, but that theres a type of button and all buttons of that type look the same.